src/github.com/getlantern/peerscanner/dsp/dsp_test.go

```diff
@@ -39,6 +40,8 @@ func TestRegister(t *testing.T) {
 		assert.NotNil(t, rec, "A new record should have been returned")
 	}
 
+	// give dnssimple enough time to complete the operation
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Ha I forgot to try run the test locally before submit code. Line 40 already verified that a new record is created, 100 millis is for DNSimple to sync internal state which should be very short I think.